lib/login_page.dart

```diff
@@ -5,6 +5,10 @@ import 'package:job_bazaar/auth.dart';
 import 'package:job_bazaar/signup_page.dart';
 import 'package:provider/provider.dart';
 
+// SingleChildScrollView(
```

Comment:
Can you, please, delete the commented lines, if it is not if they are not useable.

Reply:
sure